### PR TITLE
changed usage scenarios using jackys ideas

### PR DESCRIPTION
**Pull request by bbaillie**

### CODE REVIEW

**Review by merveshin — COMMENTED** [review]
"She will also be able to explore the historical trends of dominant sexes within a company’s industry by selecting it’s industry" for this part maybe you meant its industry ?

File: proposal.md [hunk 1]
```diff
@@ -1,23 +1,23 @@
...
...She also wants to [identify] both industry leaders in gender balance, and those with extreme inequality, to be able to learn from them . When Naomi goes on our website, she will be able to see an overview of the data including the number of jobs for which data is available and the various industries those jobs come from.  She will also be able to explore the historical trends of dominant sexes within a company’s industry by selecting it’s industry and seeing how the gender diversity has evolved over time, and she can see how it compares to other industries. She will also be able to see trends in gender balance for specific jobs, which she could see for the most balanced jobs, or for those with the most inequality on either end of the spectrum, by selecting a job from those available. When Naomi uses our website to do research for a mining company, she may find that certain engineering jobs, present in the mining company, have been historically male dominated, but that academia as an industry has been very well balanced historically. As a result, Naomi can further research how universities diversify their workforces, and apply her findings to the engineering jobs within the company.
```
Comment (jackyho112) [inline proposal.md]:
```suggestion
Naomi is a researcher in a reputable HR consulting company that often gets contracted by companies to help them grow or restructure their workforces. Achieving gender balance is crucial to many of these contracts. As a result, for many of her research, she wants to be able to [explore] historical data on the gender balance of various occupations to [gain insights] on the history of gender diversity in those industries and [devise theories] on why they were less than optimal. She also wants to [identify] industry leaders in gender balance and those with extreme inequality, to be able to learn from them. 

When Naomi goes on our website, she will be able to see an overview of the data, including the number of jobs for which data is available and the various industries of these jobs.  She will also be able to explore and compare the historical trends of dominant sexes within different industries by selecting industries and observing how their gender diversities have evolved over time. Our site also provides trends in gender balance for specific jobs, allowing Naomi to identify jobs with good and lousy gender balances from filtering available jobs.

For instance, if Naomi were tasked to research how to improve gender balance in a mining company using our website, she might find that specific mining engineering jobs have been historically male-dominated. She might also find that academia as an industry has been very well balanced historically. As a result, Naomi can further research how universities successfully diversify their workforces and apply her findings to engineering jobs within that mining company.
```

File: proposal.md [hunk 2]
```diff
@@ -1,23 +1,23 @@
...
... She also wants to [identify] both industry leaders in gender balance, and those with extreme inequality, to be able to learn from them . When Naomi goes on our website, she will be able to see an overview of the data including the number of jobs for which data is available and the various industries those jobs come from.  She will also be able to explore the historical trends of dominant sexes within a company’s industry by selecting its industry and seeing how the gender diversity has evolved over time, and she can see how it compares to other industries. She will also be able to see trends in gender balance for specific jobs, which she could see for the most balanced jobs, or for those with the most inequality on either end of the spectrum, by selecting a job from those available. When Naomi uses our website to do research for a mining company, she may find that certain engineering jobs, present in the mining company, have been historically male dominated, but that academia as an industry has been very well balanced historically. As a result, Naomi can further research how universities diversify their workforces, and apply her findings to the engineering jobs within the company.
```
Comment (jackyho112) [inline proposal.md]:
```suggestion
Naomi is a researcher in a reputable HR consulting company that often gets contracted by companies to help them grow or restructure their workforces. Achieving gender balance is crucial to many of these contracts. As a result, for many of her research, she wants to be able to [explore] historical data on the gender balance of various occupations to [gain insights] on the history of gender diversity in those industries and [devise theories] on why they were less than optimal. She also wants to [identify] industry leaders in gender balance and those with extreme inequality, to be able to learn from them. 

When Naomi goes on our website, she will be able to see an overview of the data, including the number of jobs for which data is available and the various industries of these jobs.  She will also be able to explore and compare the historical trends of dominant sexes within different industries by selecting industries and observing how their gender diversities have evolved over time. Our site also provides trends in gender balance for specific jobs, allowing Naomi to identify jobs with good and lousy gender balances from filtering available jobs.

For instance, if Naomi were tasked to research how to improve gender balance in a mining company using our website, she might find that specific mining engineering jobs have been historically male-dominated. She might also find that academia as an industry has been very well balanced historically. As a result, Naomi can further research how universities successfully diversify their workforces and apply her findings to engineering jobs within that mining company.
```

Comment (jackyho112) [reply proposal.md]: Mostly the same, just reorganized!

File: proposal.md [hunk 3]
```diff
@@ -1,23 +1,23 @@
...
...es and [devise theories] on why they were less than optimal. She also wants to [identify] industry leaders in gender balance and those with extreme inequality, to be able to learn from them. When Naomi goes on our website, she will be able to see an overview of the data, including the number of jobs for which data is available and the various industries of those jobs. She will also be able to explore and compare the historical trends of dominant sexes within different industries by selecting an industry and observing how their gender diversity has evolved over time. She will be able to observe trends in gender balance for specific jobs, which she can see for the most balanced jobs, and for those with the most inequality on either end of the spectrum. If Naomi is tasked to do research for a mining company, when she uses our website she may find that mining-specific engineering jobs have been historically male dominated. She might also find that academia as an industry has been very well balanced historically. As a result, Naomi can do further research on how universities successfully diversify their workforces and apply her findings to the engineering jobs within the mining company.
```
Comment (jackyho112) [inline proposal.md]:
```suggestion
Naomi is a researcher in a reputable HR consulting company that often gets contracted by companies to help them grow or restructure their workforces. Achieving gender balance is crucial to many of these contracts. As a result, for much of her research, Naomi wants to be able to [explore] historical data on the gender balance of various industry occupations to [gain insights] on the history of gender diversity in those industries and [devise theories] on why they were less than optimal. She also wants to [identify] industry leaders in gender balance and those with extreme inequality, to be able to learn from them. When Naomi goes on our website, she will be able to see an overview of the data, including the number of jobs for which data is available and the various industries of those jobs. She will also be able to explore and compare the historical trends of dominant sexes within different industries by selecting an industry and observing how their gender diversity has evolved. She will be able to observe trends in gender balance for specific jobs, which she can see for the most gender-balanced jobs and those with the most inequality on either end of the spectrum. If Naomi is tasked to research a mining company when she uses our website, she may find that mining-specific engineering jobs have been historically male-dominated. She might also find that academia as an industry has been very well balanced historically. As a result, Naomi can do further research on how universities successfully diversify their workforces and apply her findings to engineering jobs within the mining company.
```

**Pull request merged**